packaging/server_config.json.template

```diff
@@ -10,13 +11,18 @@
             "shared_memory_size_in_bytes": 5000000,
             "eviction_age_in_seconds": 3600
         },
+        "dns_cache_clearer_sleep_time_in_seconds": 600,
```

Comment:
i like a flat hierarchy for these settings i think.   less complicated.  less logic to parse.

Reply:
Could that lead to more confusion given we have a section dedicated to dns_cache?

Reply:
yes, probably.   bah.

Reply:
I think grouping this into the `dns_cache` and `hostname_cache` sections makes the most sense because the JSON objects already exist and it reduces the number of keywords we need to add. Plus, as we discussed, it would logically group the concepts together so that it is known that they are related.

Reply:
Go for it.

Reply:
Done